views/includes/header.js

```diff
@@ -12,6 +16,7 @@ module.exports = function(options) {
   <link rel="manifest" href="/manifest.json">
   <link rel="mask-icon" href="/safari-pinned-tab.svg" color="#64b5f6">
   <script type="text/javascript" src="/js/main.js" async></script>
+  ${structuredData}
```

Comment:
Better put the `join` here to have the relation of indention and spaces in the string.

Reply:
Done.